packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -55,7 +55,8 @@ export class InsecureUrlResolver implements IUrlResolver {
         // If hosts match then we use the local tenant information. Otherwise we make a REST call out to the hosting
         // service using our bearer token.
         if (this.isForNodeTest) {
-            const [, documentId, relativePath] = parsedUrl.pathname.substr(1).split("/");
+            const [, documentId, tmpRelativePath] = parsedUrl.pathname.substr(1).split("/");
+            const relativePath = tmpRelativePath === undefined ? "" : tmpRelativePath;
             return this.resolveHelper(documentId, relativePath, parsedUrl.search);
```

Comment:
Any chance you can see what it takes to remove this line from packages/runtime/test-runtime-utils/tsconfig.json?
        "strictNullChecks": false,

This will ensure issue like that would never happen again.

Reply:
Hi,
Sure I will take a look about strictNullChecks but it might need quite a bit of code change.
Will have another pr for strictNullChecks.